repose-aggregator/components/filters/header-translation-filter/src/main/scala/org/openrepose/filters/headertranslation/HeaderTranslationFilter.scala

```diff
@@ -101,3 +108,9 @@ class HeaderTranslationFilter @Inject()(configurationService: ConfigurationServi
     initialized = true
   }
 }
+
+object HeaderTranslationFilter {
+  private final val DefaultConfig = "header-translation.cfg.xml"
+  private final val SchemaFileName = "/META-INF/schema/config/header-translation.xsd"
+  private final val QualityReqex = ";q=\\d[^;]*".r
```

Comment:
I like how this regex is like, "the first character of the parameter value must be a digit, but after that... f*** it, anything goes!"